src/views/catalog/wizard/tabs/scripts/WizardScriptsTab.tsx

```diff
@@ -29,6 +36,20 @@ const WizardScriptsTab: WizardTab = ({ vm, updateVM }) => {
       >
         <SidebarEditorSwitch />
         <DescriptionList className="wizard-scripts-tab__description-list">
+          <DescriptionListDescription>
+            <Alert
+              className="scripts-alert"
+              title={t(
+                'Cloud-init and SSH key configurations will be applied to the VirtualMachine only at the first boot.',
+              )}
+              isInline
+              variant={AlertVariant.warning}
+            >
+              {t(
+                'Please, make sure the configuration is correct before starting the VirtualMachine.',
+              )}
+            </Alert>
+          </DescriptionListDescription>
```

Comment:
I see you use this exact alert in 2 different places, please create as a component and use it in both places